installTitan.py

```diff
@@ -5,7 +5,7 @@
 if sys.version_info < (3, 0, 0):
     raise Exception("Please use Pypy 3.")
 
-TITAN_PATH = "git+https://github.com/marshall-lab/TITAN"
+TITAN_PATH = "git+https://github.com/pph-collective/TITAN"
```

Comment:
semi-tangential question - titan is on pip now, so do we even need the github path here? or could we instead do `titan_model` (or whatever we are on pypi)